stopify-continuations/src/callcc/callcc.ts

```diff
@@ -146,7 +146,7 @@ const visitor: Visitor = {
         t.callExpression(t.identifier('require'),
           [t.stringLiteral('stopify/dist/src/stopify/compileFunction')]) :
         // provided by dist/stopify-compiler.bundle.js
-        t.memberExpression(t.identifier('stopifyCompiler'), t.identifier('module'));
```

Comment:
What's this for?

Reply:
The module exports are just properties of `stopifyCompiler`, there is no `module` property. I'm not sure how the `eval` unit tests on master are passing without this change.

Reply:
To follow up, the eval unit tests run under `node` and are compiled with `--require-runtime`, so the global free variable `stopifyCompiler` is replaced with `require('stopify/dist/src/stopify/compileFunction')`.